fc/fc_app.py

```diff
@@ -1,3 +1,4 @@
+# coding=utf-8
```

Comment:
Is there any point specifying this when using python 3?

Reply:
Nope - old habit :)